src/main/java/uk/gov/hmcts/divorce/common/notification/FinalOrderRequestedNotification.java

```diff
@@ -124,6 +119,36 @@ private Map<String, String> solicitorsFinalOrderTemplateVars(final CaseData case
         templateVars.put(SIGN_IN_URL, commonContent.getProfessionalUsersSignInUrl(caseId));
         templateVars.put(DATE_OF_ISSUE, caseData.getApplication().getIssueDate().format(DATE_TIME_FORMATTER));
 
+        applicantFinalOrderOverdueTemplateVars(templateVars, caseData);
+
         return templateVars;
     }
+
+    private Map<String, String> applicantFinalOrderTemplateVars(final CaseData caseData, final Long caseId,
+                                                                Applicant applicant1, Applicant applicant2) {
+        Map<String, String> templateVars = commonContent.mainTemplateVars(caseData, caseId, applicant1, applicant2);
+        applicantFinalOrderOverdueTemplateVars(templateVars, caseData);
+        if (YesOrNo.YES.equals(caseData.getFinalOrder().getIsFinalOrderOverdue())) {
+            templateVars.put(IS_OVERDUE, YES);
```

Comment:
if these two vars are diametrically opposed do you need them both?

Reply:
is there a way to do this in gov notify?

Reply:
I thought so - guess I was wrong